server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -172,9 +174,20 @@ export function create(
 			);
 
 			// Protocol state
-			const { sequenceNumber, values, generateToken = false } = request.body;
+			const {
+				sequenceNumber,
+				values,
+				generateToken = false,
+				isEphemeralContainer = false,
+			} = request.body;
 
 			const enableDiscovery: boolean = request.body.enableDiscovery ?? false;
+			const isEphemeral: boolean =
+				(typeof isEphemeralContainer === "undefined"
+					? false
+					: typeof isEphemeralContainer === "boolean"
+					? isEphemeralContainer
+					: isEphemeralContainer === "true") && !ignoreEphemeralFlag;
```

Comment:
Can we move this to a `getBooleanQueryParam(param: any): boolean` in services-shared? Then we can eventually share that logic in Historian and Gitrest

Reply:
Moved it to services-shared. Will use this in historian and gitrest after the bump